mobile/src/reducers/feeds/feeds.reducer.ts

```diff
@@ -45,6 +47,11 @@ export const feedsReducer: Reducer<FeedsState> = (state = initState, action: Fee
                 toDate: action.toDate,
                 fromDate: action.fromDate,
             };
+        case 'SEARCH_FEED_FILTER':
+            return {
+                ...state,
+                filter: action.filter,
```

Comment:
This state is used for people and feed at once, so when you filter feeds by some value, it also will affected on people. SEARCH_FEED_FILTER should be handled in reducer